src/components/DevTool/DevTool.tsx

```diff
@@ -83,12 +122,17 @@ export default function DevTool() {
           >
             Submit
           </Button>
-          <Collapse in={openCollapse}>
+          <Collapse in={!!stateAfter}>
             <Box mt="12px">
-              <Typography>Updated block number: {updatedBlockNum}</Typography>
               <Typography>
-                Updated block Timestamp: {updatedBlockTime}
+                Updated block number: {stateAfter?.blockNumber}
+              </Typography>
+              <Typography>
+                Updated block Timestamp: {stateAfter?.blockTimestamp}
               </Typography>
+              {stateAfter?.error ? (
+                <Typography>Error: {stateAfter?.error}</Typography>
```

Comment:
I think it would be good to give `error` color when we display error.

```
  <Typography color="error">
                  Error: {stateAfter?.error}
                </Typography>
```